services/scan/src/precinct_scanner_app.test.ts

```diff
@@ -0,0 +1,745 @@
+import { MockScannerClient, ScannerClient } from '@votingworks/plustek-sdk';
```

Comment:
This is the main change. Github collapsed it by default so commenting to ensure you don't miss it